agent/build.gradle

```diff
@@ -18,20 +23,32 @@ shadowJar {
         attributes('Automatic-Module-Name': 'reactor.blockhound')
     }
 
-    exclude 'module-info.class'
+    exclude 'META-INF/versions/9/module-info.class'
+    exclude 'META-INF/LICENSE'
+    exclude 'META-INF/NOTICE'
+    exclude 'META-INF/license/'
     exclude 'META-INF/maven/**'
     exclude 'reactor/shaded/META-INF/**'
+
+    // TODO discuss with ByteBuddy folks how to shade it
+    exclude 'win32-x86*/**'
```

Comment:
@raphw will it break ByteBuddy on Windows + JDK, or it will fallback if the DLLs are not available? Thanks!

Reply:
It's not JNI but JNA, therefore shading Byte Buddy does not cause any problems. JNA can however not be shaded. You would only need it if you wanted to use emulated attachment.

Reply:
@raphw thank you!

But, will it break if a user will have JNA on classpath (so that ByteBuddy will attempt to use the emulated attachment), but the DLLs will be missing?

Reply:
Yes, the DDL is expexted to exist if one used a self-attach with JNA on the class path on a JRE VM. It would break gracefully, though.

Reply:
Thanks a lot for handling it gracefully! 👏

Another proof that ByteBuddy is a very reliable dependency! 👍

Reply:
P.S. 

> the DDL 

I think you had a bit too much of Oracle lately 😂

Reply:
P.S.

> the DDL

@raphw I think you had a bit too much of Oracle lately 😂